### ISSUE TITLE
processes names are too long for mis and isf EMP_INIT column

### ISSUE THREAD

**Issue by kentbower** [kbower/tickettest1#2171]
_Imported from trac ticket 2171.
Created by **kbower**
Opened in trac: Feb 23, 2013, 6:56:19 AM
Last modified in trac: May 31, 2013, 2:51:13 PM_
***
All other GERS customers, even those with same version of GERS and later, have:
```
EMP_INIT     NOT NULL VARCHAR2(16)
```

While miskelly has:
```
EMP_INIT     NOT NULL VARCHAR2(8)
```

**Comment by kentbower** [comment]
_**kbower** trac comment on Feb 23, 2013, 6:57:05 AM:_

We should find out why the difference and make their column 16 if there is no good reason for it being 8.

**Comment by kentbower** [comment]
_**kbower** trac comment on Feb 25, 2013, 11:39:25 AM:_

Andrea,
We would like to fix your emp_init column to match all other GERS customers.  Is there a reason you guys have shortened this?
Thanks,
Scott

**Comment by kentbower** [comment]
_**kbower** trac comment on Mar 7, 2013, 5:54:16 AM:_

I'm going to update their column in TRAIN and see if anything bad happens...

**Comment by kentbower** [comment]
_**kbower** trac comment on Mar 7, 2013, 6:00:54 AM:_

`alter table MISC.EMP modify (EMP_INIT VARCHAR2(16));`

**Comment by kentbower** [comment]
_**kbower** trac comment on Apr 26, 2013, 10:31:16 AM:_

Ivan smith also.

**Comment by kentbower** [comment]
_**kbower** trac comment on Apr 26, 2013, 10:37:57 AM:_

Also updated isf TRAIN today.

**Comment by kentbower** [comment]
_**kbower** trac comment on Apr 26, 2013, 10:46:00 AM:_

in another month or so, I'll be adding data update to `mis.py` and `isf.py` to modify this column length on live.

**Comment by kentbower** [comment]
_**kbower** trac comment on May 31, 2013, 2:51:13 PM:_

(Changeset [[4199]](../commit/814c382270ccb218e8e0319cc339ca3e3edbd874))
* fixes #2171, increase MISC.EMP EMP_INIT column size to varchar(16) #isf, #mis

**Comment by kentbower** [comment]
_**kbower** trac status change on May 31, 2013, 2:51:13 PM:_ **closed**
